README.md

```diff
@@ -304,6 +304,43 @@ If any of these steps error with, `file table overflow`, you are probably on a m
 which has a very low limit on max open files. Run `ulimit -Sn 1024` and try again.
 You'll need to do this in each new terminal you open before building Riot.
 
+Using Docker image for development
```

Comment:
There should be a mention that this won't work on Windows due to an OS limitation. Windows file watches don't fire to containers because the host doesn't proxy the watches to the virtual machine. It's a known problem with Vagrant too, unfortunately.

We don't have many Windows developers, but we do have enough where this could easily trip them up. It'll start up perfectly fine, but changes they make won't rebuild.

Reply:
While it is true for the default behavior, I believe it worked for me in the past with `watchOptions: { poll: true }` i.e. making WebPack webserver to check the changes instead of e.g. `inotify` or alternatives. Although it doesn't mean the problem is not present on Windows, it should work as a workaround.

I don't have Windows though, so I can't test it, but I'll at least add it as a note to `README.md`